steps/validate.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Show Action Version
+echo "Using Rainforest GitHub Action v${RF_ACTION_VERSION}"
+
+# Ensure results directory is there
+mkdir -p results/rainforest
+
+echo "::add-mask::${RF_TOKEN}"
+
+# Define error helper
+error () {
+  echo "::error ::${1}"
+  echo "error=${1}" >> "$GITHUB_OUTPUT"
+  exit 1
+}
+
+# Validate token
+if [ -z "${RF_TOKEN}" ] ; then
+  error "Token not set"
```

Comment:
Pedantry: this is actually testing if the variable has a zero length value, `test -v` tests if a variable is set.

Reply:
Right, but since tokens can't be zero-length, if the _variable_ is set to the empty string, then the _token_ isn't actually set